### ISSUE TITLE
Labels for ID and Ratings columns are abbreviated due to limited widths

### ISSUE THREAD

**Issue by kuanghe-dev** [zerotrac/leetcode_problem_rating#3]
Is it possible to increase the width of both the ID and Ratings columns so that the full labels get shown? See this screenshot.

![image](https://user-images.githubusercontent.com/84287479/148611784-d9e3d709-059b-46be-95bc-8b45f5ff0683.png)

While you are at it, perhaps you could increase **width: 60%** in the index.html a bit as well?

```
<div id="wrapper" style="width: 60%; margin: auto"></div>
```

**Comment by zerotrac** [comment]
Making the table 60% width achieves the best impression on my monitor, so I may not modify that.

But there is a problem on the ID label problem (but I cannot reproduce the Rating label problem). I will add 1% width to the ID column and it now looks like below:

![image](https://user-images.githubusercontent.com/15974765/148617562-9422690d-481e-48ba-96a4-0c824a10d1b9.png)

**Comment by kuanghe-dev** [comment]
Now I see why it's been less of a problem for you. The abbreviation starts to take place when the width of the browser window gets smaller. The screenshot I provided above was taken when I had resized the browser window to 1920x1080 (only using part of the screen). The problem got accentuated for me since my display setting on Windows scaled everything up by 150%.

Overall, it's not a huge problem. Thanks for making the web page and maintaining the database. It's very useful!
